### ISSUE TITLE
Add a data map to the "finished" page

### ISSUE THREAD

**Issue by ThomasLaPiana** [ethyca/fides#600]
### Is your feature request related to a specific problem?

On the final page of the Config UI workflow, we should show users a basic data map in addition to the raw yaml

### Describe the solution you'd like

_A description of what you want to happen._

### Describe alternatives you've considered, if any

_A description of any alternative solutions or features you've considered._

### Additional context

dependent on #599

**Comment by ThomasLaPiana** [comment]
closing, completed via another ticket/stale
